app/models/project.rb

```diff
@@ -32,7 +32,16 @@ def has_reducers?
   end
 
   def public_data?(type)
-    public_reductions?
+    case type
+    when 'extracts'
```

Comment:
Minor not blocking, the default else clause will catch this vs explicit extra comparison. Maybe leave it out till the different behaviour is needed.

Reply:
i wanted to be explicit about this because the behavior is different from workflows (where we check for `public_extracts?`). I think the clarity is important here.